aces/Loris
# [session] Add date of status change to session table

## Brief summary of changes
This PR adds a column in the session table for Date_status_change that is updated when the status of a visit is set to 'Pass', 'Failure', 'Withdrawal', or 'In Progress'


#### Testing instructions (if applicable)

1. Source patch in this PR
2. Start a new visit
3. When the visit status is changed, the date column should be updated accordingly

#### Link(s) to related issue(s)

* No issue was created for this, but it was discussed as an action item in a Loris meeting (See agenda for October 18, 2022)
